interactive/__init__.py

```diff
@@ -0,0 +1,28 @@
+from attrs import define
+
+
+@define
+class Codelist:
+    label: str
+    slug: str
+    type: str  # noqa: A003
+
+    # TODO: what are these again?
+    path: str | None = None
+    description: str | None = None
+
+
+@define
+class Analysis:
```

Comment:
It is, the Django Form will be too and I just don't have a nice way to handle all of that yet.  I have a feeling we will need to package these up together somehow and I wouldn't be opposed to reaching for something like pydantic since it did both and we're only using the Django form for validation (and haven't even tackled sending failures back yet!)

Are you happy to leave this as-is until we have a better understanding of what's coming?  I can put a comment in to the effect of what I've said above.

Reply:
Yeah, comment would be good

Reply:
And pydantic does feel like a good fit to me